test/dymaptic.GeoBlazor.Core.Test.Blazor.Shared/Components/FeatureLayerViewTests.razor

```diff
@@ -58,4 +58,194 @@
         Assert.IsNotNull(result);
         Assert.IsTrue(result.Features!.Count > 0);
     }
+
+  [TestMethod]
+    public async Task FeatureLayerView_PropertyRoundtrips_And_Queries(Action renderHandler)
+    {
+        FeatureLayer? layer = null;
+        FeatureLayerView? layerView = null;
+
+        void OnLayerViewCreate(LayerViewCreateEvent e)
+        {
+            if (e.Layer is FeatureLayer && e.LayerView is FeatureLayerView flv)
+            {
+                layerView = flv;
+            }
+        }
+
+        // In-memory layer for deterministic queries/highlights
+        var graphics = new List<Graphic>
+        {
+            new Graphic(
+                new Point(0, 0),
+                attributes: new AttributesDictionary(new Dictionary<string, object?> { ["OBJECTID"] = 1 })
+            ),
+            new Graphic(
+                new Point(1, 1),
+                attributes: new AttributesDictionary(new Dictionary<string, object?> { ["OBJECTID"] = 2 })
+            )
+        };
+
+        AddMapRenderFragment(
+            @<MapView class="map-view" OnViewRendered="renderHandler" OnLayerViewCreate="OnLayerViewCreate">
+                <Map>
+                    <Basemap><BasemapStyle Name="BasemapStyleName.ArcgisTopographicBase" /></Basemap>
+                    <FeatureLayer @ref="layer"
+                                  Source="graphics"
+                                  OutFields="@(["*"])"
+                                  SpatialReference="SpatialReference.Wgs84"
+                                  GeometryType="FeatureGeometryType.Point"
+                                  ObjectIdField="OBJECTID" />
+                </Map>
+            </MapView>);
+
+        await WaitForMapToRender();
+
+        // Wait for the FeatureLayerView
+        for (var i = 0; layerView is null && i < 100; i++) { await Task.Delay(50); }
```

Comment:
We have `WaitForLayerToRender` built in to the test runner.

```csharp
LayerViewCreateEvent createEvent = await WaitForLayerToRender<FeatureLayer>();
```

Whenever possible, try to avoid using the random timeouts. I know it's not always possible, but since we have this one it is.